src/components/UserDetail/UserDetail.js

```diff
@@ -6,6 +6,7 @@ import {
   FormattedDate,
   FormattedMessage,
 } from 'react-intl';
+import classNames from 'classnames';
```

Comment:
@zburke do we need to bring extra dependency for combining two classes together? 
Wouldn't this 

````js
className={`floatEnd ${css.userPlaceholder}`}` 
````

work?